pkg/kev/config/extensions_test.go

```diff
@@ -124,4 +171,43 @@ var _ = Describe("Extentions", func() {
 		})
 	})
 
+	Describe("Merge", func() {
+		It("merges target into base", func() {
+			k8sBase := config.DefaultK8SConfig()
+			var k8sTarget config.K8SConfiguration
+			k8sTarget.Workload.Replicas = 10
+			k8sTarget.Workload.LivenessProbe.Type = config.ProbeTypeNone.String()
+
+			expected := k8sBase
+			expected.Workload.Replicas = 10
+			expected.Workload.LivenessProbe.Type = config.ProbeTypeNone.String()
+
+			result, err := k8sBase.Merge(k8sTarget)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(result).To(BeEquivalentTo(expected))
+		})
+
+		When("configs are empty/invalid", func() {
+			It("returns default when map is empty", func() {
+				extensions := make(map[string]interface{})
```

Comment:
These 3 lines belong to `BeforeEach` as a stage setup for test cases in this context block. Similar comment goes to the the other case below which would be best to be encapsulated in a separate context.